bundles/framework/layerlist/resources/scss/style.scss

```diff
@@ -1,5 +1,3 @@
-$filter-border-color: #a4a4a4;
-
 @keyframes blinking {
```

Comment:
These seem to repeat as a styled-component. Could these be removed as well?

Reply:
Unfortunately no, these are for the Tile.